### DISCUSSION TITLE
When I build the project I get an error

### DISCUSSION THREAD

**Discussion by maicFir** [wagmi-dev/abitype#222] [Q&A]
./node_modules/.pnpm/registry.npmmirror.com+abitype@0.9.8_typescript@4.9.3/node_modules/abitype/dist/types/human-readable/formatAbi.d.ts:18:35
Type error: Type parameter declaration expected.

  16 |  * @returns Human-readable ABI
  17 |  */
> 18 | export declare function formatAbi<const TAbi extends Abi | readonly unknown[]>(abi: TAbi): FormatAbi<TAbi>;
     |                                   ^
  19 | //# sourceMappingURL=formatAbi.d.ts.map
- info Checking validity of types ... ELIFECYCLE  Command failed with exit code 1.
- 
<img width="1036" alt="Snipaste_2024-01-12_20-20-09" src="https://github.com/wevm/abitype/assets/21973334/f8373a69-faa4-4742-8813-68325c977529">

my abitype is 0.10.3

**Comment by tmm** [comment]
ABIType requires `typescript@>=5.0.4`, you have `typescript@4.9.3`.
